content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+
+### Manual Root Certificate Rotation
+
+#### Tresor
+
+>  WARNING: Rotating root certificates will incur downtime between any services as they transition their mTLS certificate from one issuer to the next.
+
+The self-signed root certificate, which is created via the Tresor package within OSM, will expire in a decade. To rotate the root certificate, the following steps should be followed:
+
+1. Delete the `osm-ca-bundle` certificate in the OSM namespace
+   ```console
+   export osm_namespace=osm-system # Replace osm-system with the namespace where OSM is installed
+   kubectl delete secret osm-ca-bundle -n $osm_namespace
+   ```
+
+2. Restart the control plane components
+   ```console
+   kubectl rollout restart deploy osm-controller -n $osm_namespace
+   kubectl rollout restart deploy osm-injector -n $osm_namespace
+   kubectl rollout restart deploy osm-bootstrap -n $osm_namespace
+   ```
+
+When the components gets re-deployed, you should be able to eventually see the new `osm-ca-bundle` secret in `$osm_namespace`:
+
+```console
+kubectl get secrets -n $osm_namespace
+```
+
+```
+NAME                           TYPE                                  DATA   AGE
+osm-ca-bundle                  Opaque                                3      74m
+```
+
+The new expiration date can be found with the following command:
+
+```console
+kubectl get secret -n $osm_namespace $osm_ca_bundle -o jsonpath='{.data.ca\.crt}' |
+    base64 -d |
+    openssl x509 -noout -dates
+```
+
+For Envoy services and validation certificates to be rotated the data plane components must be restarted.
+
+#### Other certificate providers
+The process of rotating the root certificate will be different for certificate providers other than Tresor. For Hashicorp Vault and cert-manager.io, users will need to rotate the root certificate themselves outside of OSM.
+
+### Certificate Rotation with MeshRootCertificate
+
+>  WARNING:  This feature is currently in preview and requires the `EnableMeshRootCertificate` feature flag enabled in the MeshConfig.  
+
+The `MeshRootCertificate` is introduced in OSM v1.3. This custom resource can be enabled by using the `EnableMeshRootCertificate` feature flag when deploying OSM.
+
+```console
+osm install --set="osm.featureFlags.enableMeshRootCertificate=true"
+```
+
+> Important: If OSM is installed with `EnableMeshRootCertificate` disabled, but the feature flag is then later enabled in the MeshConfig, the control plane components must be restarted. This allows the flag to be picked up and begin using the `MeshRootCertificate` for certificate management. There will be no change in the root certificate, but it will now be managed by the `MeshRootCertificate`. 
+
+### MeshRootCertificate
```

Comment:
I think this section should actually be in a separate doc that explains MeshRootCertificate because I think certificate-rotation.md is mainly focused on how to do the rotation given each certificate provider or with MRC